applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -151,6 +149,7 @@ static struct {
 		uint16_t ctr; /* Count func calls. Used for collecting data points and waiting */
 		int32_t sum_err_dly_us;
 		uint32_t pres_delay_us;
+		bool audio_comp_enabled;
```

Comment:
`Audio comp` is a quite generic name. Can you specify?

Reply:
After reading the entire PR again: Since this variable is within the `pres_comp` struct, just `enabled` will do as the name. Make the same change to the `drift_comp` struct.

Reply:
Changed to enabled